dbscripts/xml/upgrade/3.1.1_update.xml

```diff
@@ -38,4 +38,15 @@
 		<query>UPDATE journal_settings SET setting_name = 'subjectsEnabledWorkflow' WHERE setting_name = 'subjectEnabledWorkflow'</query>
 		<query>UPDATE journal_settings SET setting_name = 'subjectsRequired' WHERE setting_name = 'subjectRequired'</query>
 	</sql>
+	<!-- pkp/pkp-lib#2894 Set 0 review ratings to NULL -->
+	<sql>
+		<query>UPDATE review_assignments ra SET ra.quality = NULL WHERE ra.quality = 0</query>
+	</sql>
+	<!-- pkp/pkp-lib#2894 Update old localised user gossip entries -->
+	<sql>
+		<query driver="mysql">UPDATE users u, (SELECT user_id, GROUP_CONCAT(DISTINCT setting_value SEPARATOR ' ') AS groupedGossips FROM user_settings WHERE setting_name = 'gossip' AND setting_value &lt;&gt; '' GROUP BY user_id) us SET u.gossip = us.groupedGossips WHERE us.user_id = u.user_id</query>
+		<query driver="mysqli">UPDATE users u, (SELECT user_id, GROUP_CONCAT(DISTINCT setting_value SEPARATOR ' ') AS groupedGossips FROM user_settings WHERE setting_name = 'gossip' AND setting_value &lt;&gt; '' GROUP BY user_id) us SET u.gossip = us.groupedGossips WHERE us.user_id = u.user_id</query>
+		<query driver="postgres7">UPDATE users SET citations = us. groupedGossips FROM (SELECT user_id, string_agg(DISTINCT setting_value, ' ') AS groupedGossips FROM user_settings WHERE setting_name = 'gossip' AND setting_value &lt;&gt; '' GROUP BY user_id) AS us WHERE us.user_id = users.user_id</query>
```

Comment:
There were a couple of typos here that I fixed -- `us. groupedGossips` (note the space), `citations` rather than `gossip`. (Also fixed in OMP.)